drivers/w1/w1_ds2484.c

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2022 Caspar Friedrich <c.s.w.friedrich@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "w1_ds248x.h"
+
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/w1.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/pm/device.h>
+#include <zephyr/zephyr.h>
+
+#define DT_DRV_COMPAT maxim_ds2484
+
+LOG_MODULE_REGISTER(ds2484, CONFIG_W1_LOG_LEVEL);
+
+struct ds2484_config {
+	struct w1_master_config w1_config;
+	const struct i2c_dt_spec i2c_spec;
+	const struct gpio_dt_spec slpz_spec;
```

Comment:
No `const` is needed in the config struct. (Only if they were pointers to a const struct)